runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -41,8 +42,14 @@
     @Getter
     Multimap<UUID, IStreamView> streamCache = Multimaps.synchronizedMultimap(HashMultimap.create());
 
+    /**
+     * The compression codec used to compress a stream's append payload
+     */
+    final Codec.Type codec;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4705140)